samples/sensor/mcp9808/boards/nucleo_l053r8.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2019, Linaro Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
+	mcp9808@18 {
+		compatible = "microchip,mcp9808";
+		reg = <0x18>;
+		int-gpios = <&gpioa 8 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+		label = "MCP9808";
+	};
+};
```

Comment:
Please add this board to the `sample.yaml`

Reply:
The board has now been added to `sample.yaml`